Update _1_Calibrate_Z_Probe.g

quarky42 opened the pull request:
This was the message I meant to come back and update.  On the Quad, retracting the z probe isn't optional.

quarky42 committed:
Update _1_Calibrate_Z_Probe.g

This was the message I meant to come back and update.  On the Quad, retracting the z probe isn't optional.

ericl-m3d commented:
Quarky42,

Thanks for your suggestions. There is a situation where retracting the probe is not optional (moving onto the 2nd macros). But, there are occasions when users skips the 2nd ("_2_Set_Z_Endstop_Height") macros and move straight to the 3rd macros. There's no need to retract probes at that point. 

Additionally, there is enough space during 2nd macros operation to retract the probe (when touching the nozzle with the bed), if they fail to do so before. 

-- M3D Team

quarky42 commented:
Thank you for the explaination.  That makes sense.

I thought this might be a carryover from something I introduced in an
earlier update in some of the wording I tweaked when I was adding more room
for being able to get to the probe itself.

I added a magnet to the right side of my ProMega print carriage.  It is
easier to store the probe switch there.

On Mon, May 6, 2019, 1:26 PM ericl-m3d <notifications@github.com> wrote:

> Quarky42,
>
> Thanks for your suggestions. There is a situation where retracting the
> probe is not optional (moving onto the 2nd macros). But, there are
> occasions when users skips the 2nd ("_2_Set_Z_Endstop_Height") macros and
> move straight to the 3rd macros. There's no need to retract probes at that
> point.
>
> Additionally, there is enough space during 2nd macros operation to retract
> the probe (when touching the nozzle with the bed), if they fail to do so
> before.
>
> -- M3D Team
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/PrintM3D/Promega/pull/48#issuecomment-489744088>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/ALQQATBXF6UD3NX5NTJHXODPUCA5NANCNFSM4HKZ3LQA>
> .
>
